src/GrpcProtos/capture.proto

```diff
@@ -349,6 +349,11 @@ message CGroupMemoryUsage {
   int64 mapped_file_bytes = 5;
   int64 pgfault = 6;
   int64 pgmajfault = 7;
+  int64 unevictable_bytes = 8;
```

Comment:
Maybe a link to some documentation is enough here for every field, or a short comment in each one.

Reply:
Thanks for the suggestion. I added the link to the documentation to the memory protos.